cli/rmi.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// RmiCommand use to implement 'rmi' command, it remove one or more images by reference
+type RmiCommand struct {
+	baseCommand
+	force bool
+}
+
+// Init initialize rmi command
+func (cc *RmiCommand) Init(c *Cli) {
+	cc.cli = c
+	cc.cmd = &cobra.Command{
+		Use:   "rmi image ",
+		Short: "Remove one or more images by reference",
```

Comment:
Please add Long and Example here like create command https://github.com/alibaba/pouch/blob/master/cli/create.go#L25-L34:
```
cc.cmd = &cobra.Command{
		Use:   "create [image]",
		Short: "Create a new container with specified image",
		Long:  createDescription,
		Args:  cobra.MinimumNArgs(1),
		RunE: func(cmd *cobra.Command, args []string) error {
			return cc.runCreate(args)
		},
		Example: createExample(),
	}
```
These things are merged this morning and useful to auto generate markdown document, so you may have no idea about that now.